internal/metrics/metrics.go

```diff
@@ -286,6 +298,44 @@ func mustRegisterUpnessMetrics(prefix string, meter metric.Meter) ([]metric.Obse
 	}, observeFunc
 }
 
+func observeTpuChipCount(o metric.Observer, tpuChipCountMetric metric.Int64ObservableGauge, schJobs map[string]records.ScheduledJob, jobSetNodesUp map[string]records.Upness, nodepools map[string]records.Upness) {
+	log.V(3).Info("observeTpuChipCount")
+	// create map of {jobsetName} -> [leaderjobNames, ...]
+	jobsByJobset := map[string]map[string]string{}
+	for npName, sch := range schJobs {
```

Comment:
Lets avoid using the scheduled jobs section of the report as a vehicle or reporting information about the JobSet. I would prefer to refactor this into the summaries for JobSets and NodePools to keep with the current patterns.

IMO, the current feature request is to add the following metrics: 1. Chip count for a given JobSet and 2. Chip count for a given NodePool.

NOTE: The lifecycle of the scheduling part of the report is not 1:1 to the lifecycle of the JobSet/NodePool.